client/profile-wizard/steps/product-types/index.js

```diff
@@ -100,15 +105,17 @@ export class ProductTypes extends Component {
 
 		return (
 			<div className="woocommerce-profile-wizard__product-types">
-				<H className="woocommerce-profile-wizard__header-title">
-					{ __(
-						'What type of products will be listed?',
-						'woocommerce-admin'
-					) }
-				</H>
-				<H className="woocommerce-profile-wizard__header-subtitle">
-					{ __( 'Choose any that apply' ) }
-				</H>
+				<div className="woocommerce-profile-wizard__step-header">
+					<Text variant="title.small" as="h2">
+						{ __(
+							'What type of products will be listed?',
+							'woocommerce-admin'
+						) }
+					</Text>
+					<Text vairant="body">
```

Comment:
Typo here

Reply:
wow thanks for spotting this.

Reply:
Fixed here: https://github.com/woocommerce/woocommerce-admin/pull/5059/commits/232b98c1e2656973e4654ed562b72592d9f52763